dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/interruptedjobwindow/RecoverInterruptedJob.java

```diff
@@ -54,48 +56,54 @@ public UUID getUuid() {
 
     @Override
     public void executeJob() throws Exception {
+        final Calendar jobStartTime = Calendar.getInstance();
         try {
             final FilesAndFolderMap filesAndFolderMapMap = endpointInfo.getJobIdAndFilesFoldersMap().get(uuid.toString());
             final Ds3Client client = endpointInfo.getClient();
             final String date = DateFormat.formatDate(new Date());
             updateTitle("Recovering " + filesAndFolderMapMap.getType() + " job of " + endpointInfo.getEndpoint() + " " + date);
             Platform.runLater(() -> endpointInfo.getDeepStorageBrowserPresenter().logText("Recovering " + filesAndFolderMapMap.getType() + " job of " + filesAndFolderMapMap.getDate(), LogType.INFO));
             final Ds3ClientHelpers helpers = Ds3ClientHelpers.wrap(client, 100);
-
             Ds3ClientHelpers.Job job = null;
             Path fileTreeModel = null;
-
             final Map<String, Path> files = filesAndFolderMapMap.getFiles();
             final Map<String, Path> folders = filesAndFolderMapMap.getFolders();
-
             final long totalJobSize = filesAndFolderMapMap.getTotalJobSize();
-
             if (filesAndFolderMapMap.getType().equals("PUT")) {
                 job = helpers.recoverWriteJob(uuid);
                 updateMessage("Initiating transfer to " + job.getBucketName());
 
             } else if (filesAndFolderMapMap.getType().equals("GET")) {
                 job = helpers.recoverReadJob(uuid);
                 fileTreeModel = Paths.get(filesAndFolderMapMap.getTargetLocation());
-
                 updateMessage("Initiating transfer from " + job.getBucketName());
             }
-
             final AtomicLong totalSent = new AtomicLong(0L);
-
-
             job.attachDataTransferredListener(l -> {
                 updateProgress(totalSent.getAndAdd(l) / 2, totalJobSize);
                 totalSent.addAndGet(l);
             });
-
             job.attachObjectCompletedListener(s -> {
+                final Calendar currentTime = Calendar.getInstance();
```

Comment:
The new java 8 time libraries should be used rather than the old pre java 8 time libraries.